### DISCUSSION TITLE
Why's it called PnP ?

### DISCUSSION THREAD

**Discussion by spamjoes** [pnp/pnpcore#1492] [General] [answered]
l'm trying to find an answer for management why we're using a tool called PnPCore.  I think PnP brings memories of Plug-n-Play, which has been fraught with security issues over its existence.  What's the PnP acronym mean in this instance and how exactly does it relate to Sharepoint / Teams / MS 365?  Thanks, any insight is appreciated... I can't seem to find an answer to this one.  Plus, I'm also now curious about it myself.

**Comment by Barbarur** [comment, marked as answer]
Patterns and Practices

**Reply by jansenbe** [reply]
This is open-source project is one of the many under the Patterns and Practices Microsoft 365 community, see https://pnp.github.io/ to learn more.

**Reply by spamjoes** [reply]
Thank you both!
